README.md

```diff
@@ -34,9 +38,9 @@ Also available per the PSR-3 standard are:
 * `debug()`
 * `log()`
 
-For a full list, see the PSR-3 documentation](http://www.php-fig.org/psr/psr-3/).
+For a full list, see the [PSR-3 documentation](http://www.php-fig.org/psr/psr-3/).
 
-It's a good idea to throw these in your user code where appropriate. Logging can be very useful for debugging. When using the logger in your classes, you need not keep going through `Injector` to get a logger instance. A cleaner approach is to inject the logger as a dependency.
+It's a good idea to throw these in your user code where appropriate, as logging is very useful for debugging. When using the logger in your classes, you need not keep going through `Injector` to get a logger instance. A cleaner approach is to inject the logger as a _dependency_.
```

Comment:
While I agree, you can only really use low log levels (debug, info) without the default log handler in SS catching and displaying them, which makes your website unusable. Higher levels should be used sparingly in cases where they're necessary, e.g. only log a critical message when there's a critical issue in your website, knowing that it will probably stop your website from behaving correctly at the same time